### DISCUSSION TITLE
Problema attivazione tapparella

### DISCUSSION THREAD

**Discussion by mirtillo64** [indomus/forum#3182] [GENERALE - Discussioni generiche]
Ciao a tutti,
uso un’automazione che mi chiude la tapparella della mia camera, all’uscita della casa e un’altra che la apre del 45 % all’entrata, il tutto funziona.
Il problema è che se apro la tapparella dalla Dashboard o dal pulsante fisico (interruttore a muro) si apre al 45% come se prendesse il comando dall’ automazione .
Qualcuno mi può dare una mano ?
Dimenticavo uso uno Shelly plus 2pm.
Ringrazio anticipatamente, sono alle prime armi saluti

**Comment by michele-diurno** [comment]
La butto li, forse devi rifare la calibrazione della tapparella e riprovare.

**Reply by mirtillo64** [reply]
Già fatto non è quello grazie comunque

**Comment by michele-diurno** [comment]
Ricontrolla la logica dell'automazione, forse la chiusura completa della tapparella è impostata al 45% anzichè a 0%? Strano che dal pulsante fisico non chiude completamente, il tasto non è collegato a nessuna automazione? Sto facendo dei ragionamenti/ipotesi per cercare una soluzione ...

**Reply by mirtillo64** [reply]
Ho controllato ma per l'uscita da casa nell'automazione è chiudere e basta senza percentuali e funziona.

**Reply by michele-diurno** [reply]
Mi arrrendo. Aspetta qualcuno che ha più esperienza. Anche io sono circa 3 mesi che smanetto con HA ...

**Reply by mirtillo64** [reply]
Penso di aver risolto,
Siccome caricavo i vari Shelly anche su HomeKit, mi esportava la tapparella con l’impostazione dell’automazione. Ho cancellato tutto da HomeKit e reistallato uno a uno i vari dispositivi. Almeno questo penso fosse il problema comunque fatta la cosa ora mi funziona tutto come prima
Ciao

**Comment by michele-diurno** [comment]
Ben fatto chi l'ha dura vince!!!

**Reply by mirtillo64** [reply]
Grazie
